### ISSUE TITLE
Colours aren't shown in ansi-term in Emacs

### ISSUE THREAD

**Issue by Wilfred** [Wilfred/difftastic#177]
Bold works fine, inspecting it shows:

```
There are text properties here:
  font-lock-face       ((:foreground "#eeeeec" :background "#2e3434" :inverse-video nil) :inherit term-bold)
  fontified            t
```

but no colours are shown for the output, even if using `--color=always`. Strangely cargo output colours are fine.

Could experiment with inspect TERM (looking for e.g. xtern-256color or eterm-color) but that doesn't help if `always` doesn't work.

Note that eshell doesn't even get bold styling.

**Comment by cannero** [comment]
Made a short test, with `--color=always` it works for eshell for difftastic and cargo on windows with emacs 29. Without the color option I also don't get colors.

![image](https://user-images.githubusercontent.com/11857491/169135508-2120a49b-8f5f-48dd-af85-0be96b3c3929.png)

**Comment by Wilfred** [comment]
Yeah, colours look correct in eshell now, although some are missing in ansi-term still.

**Comment by Narcha** [comment]
Probably related:
If I have difftastic configured as my default difftool in git like this:
```gitconfig
[diff]
	external = difft
```
Then, diffs displayed by [paru](https://github.com/morganamilo/paru) (which uses git's configured diff) are also missing colors (making them almost entirely useless).

**Comment by Gaweringo** [comment]
Hi, @rmburg. Have you found a solution or workaround to this problem?
Because this problem has been bothering be for a while now as well. My current solution is to net set difftastic as `diff.external`, but instead use aliases that have `-c diff.external=difft` set. Have you found anything better?

**Comment by rmburg** [comment]
If I remember correctly I set

```gitconfig
[diff]
        external = difft --color=always
```

In my .gitconfig, and use git diff --no-ext-diff when needed
